src/ServiceControl.Audit/Auditing/AuditIngestionComponent.cs

```diff
@@ -12,10 +13,12 @@
     using NServiceBus.Logging;
     using NServiceBus.Transport;
     using Raven.Client;
+    using ServiceControl.Infrastructure.Metrics;
 
     class AuditIngestionComponent
     {
         static ILog log = LogManager.GetLogger<AuditIngestionComponent>();
+        static readonly long FrequencyInMilliseconds = Stopwatch.Frequency / 1000;
```

Comment:
Isn't the stopwatch frequency dependent on the resolution of the operating system?

https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.frequency?view=net-5.0#remarks

so under high-resolution mode this value might differ right?

Reply:
Will still be fine. According to the docs it `Gets the frequency of the timer as the number of ticks per second. This field is read-only.` So if I want to get the number of ticks per millisecond, then I just divide by 1000.